dvc_http/__init__.py

```diff
@@ -88,12 +88,14 @@ def _prepare_credentials(self, **config):
         # The connector should not be owned by aiohttp.ClientSession since
         # it is closed by fsspec (HTTPFileSystem.close_session)
         client_kwargs["connector_owner"] = False
-        client_kwargs["connect_timeout"] = config.get(
-            "connect_timeout", self.REQUEST_TIMEOUT
-        )
-        client_kwargs["sock_connect_timeout"] = config.get(
-            "sock_connect_timeout", self.REQUEST_TIMEOUT
+
+        connect_timeout = config.get("connect_timeout", self.REQUEST_TIMEOUT)
+        sock_connect_timeout = config.get(
+            "sock_connect_timeout", connect_timeout
         )
+
+        client_kwargs["connect_timeout"] = connect_timeout
+        client_kwargs["sock_connect_timeout"] = sock_connect_timeout
         client_kwargs["sock_read_timeout"] = config.get(
             "sock_read_timeout", self.REQUEST_TIMEOUT

```

Comment:
WDYT about renaming this to `read_timeout` as well?

Reply:
Agreed